imageio/imageio-core/src/test/java/com/twelvemonkeys/imageio/stream/BufferedChannelImageInputStreamTest.java

```diff
@@ -249,12 +248,7 @@ public void testReadShort() throws IOException {
                 assertEquals(buffer.getShort(), stream.readShort());
             }
 
-            assertThrows(EOFException.class, new ThrowingRunnable() {
-                @Override
-                public void run() throws Throwable {
-                    stream.readShort();
-                }
-            });
+            assertThrows(EOFException.class, () -> stream.readShort());
```

Comment:
I first though of changing them, the IDE itself show to replace lambda with method reference, but there is an assertion 
```
assertThrows(EOFException.class, () -> stream.readFully(new byte[1]));
``` 
for this method alone, we need to use lambda expression.

I believe it will be more readable if common cases follows the same pattern. 
No problem, I will change that to method reference in all the similar occurrences.

Reply:
Updated to method reference instead of lambda expression(for similar occurrences too)